packages/contracts/src/errors.ts

```diff
@@ -76,6 +79,14 @@ export const ErrorsPrefix: ErrorsPrefixType = {
   "#RAO": "renounceAssetOwnership",
   "#PNO": "proposeNewOwner",
   "#RO": "renounceOwnership",
+  "#RC_ARF": "routerContract_addRelayerFee",
+  "#RC_RRF": "routerContract_removeRelayerFee",
+  "#RC_RL": "routerContract_removeLiquidity",
+  "#RC_P": "routerContract_prepare",
+  "#RC_F": "routerContract_fulfill",
+  "#RC_C": "routerContract_cancel",
+  "#RF_I": "routerFactory_init",
+  "#RF_CR": "routerFactory_createRouter",
```

Comment:
This is using a slightly different error code format, i see the benefits but i'm a sucker for consistency.

Reply:
yeah I don't like it. With error codes, won't we know what contract the error comes from? Even if there's a call to another contract within that method (in the case of Router.sol)? if that's the case, this naming seems redundant